packages/components/datetime/src/RelativeDateTime.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import dayjs from 'dayjs';
+import utcPlugin from 'dayjs/plugin/utc';
+import relativeTime from 'dayjs/plugin/relativeTime';
+import calendarPlugin from 'dayjs/plugin/calendar';
+import isTodayPlugin from 'dayjs/plugin/isToday';
+dayjs.extend(utcPlugin);
+dayjs.extend(relativeTime);
+dayjs.extend(calendarPlugin);
+dayjs.extend(isTodayPlugin);
+
+export interface RelativeDateTimeProps {
+  /**
+   * The date that will be displayed. It accepts a JS Date, an ISO8601 Timestamp string, or Unix Epoch Milliseconds number
+   */
+  date: Date | string | number;
+  /**
+   * If a value is passed to baseDate, then the component will compare both dates and return the time between them.
+   * If no value is passed then the date will be compared to "now"
+   *
+   * @default "Now"
+   */
+  baseDate?: Date | string | number;
+  /**
+   * Sets the date to be relative only if it is in the current week
+   */
+  isRelativeToCurrentWeek?: boolean;
+  className?: string;
+  testId?: string;
+}
+
+const now = new Date();
```

Comment:
:important:

`now` should be calculated at the moment of rendering. move it to the component.

Reply:
🤦